### PR TITLE
feat: create privacy policy

### PR DESCRIPTION
**Pull request by wldeh**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch
  
  

---

### Description

This PR adds a detailed privacy policy at /legal/privacy-policy UR

### Related issue

- #44

### CODE REVIEW

**Comment by andrewtavis** [comment]
Thanks for your work here as well, @wldeh! I’ll check all this when you’ve finalized them :)

**Comment by wldeh** [comment]
> Thanks for your work here as well, @wldeh! I’ll check all this when you’ve finalized them :)

For sure! Yeah I'm not sure if there were any plans to set up a Figma page for this project. Regarding the policy contents I've committed here, there's definitely room for improvement. I would appreciate any guidance or direction you may have. Looking forward to your feedback!

**Comment by andrewtavis** [comment]
Thanks, @wldeh! There were no Figma files for this page specifically, but I'll likely style it the same as the about page once the text contents are merged. We can also do an actual `PRIVACY.txt` file for this as well like in Scribe :) I'll check this later!

**Comment by wldeh** [comment]
> Thanks, @wldeh! There were no Figma files for this page specifically, but I'll likely style it the same as the about page once the text contents are merged. We can also do an actual `PRIVACY.txt` file for this as well like in Scribe :) I'll check this later!

Appreciate the update @andrewtavis! Are we planning to put the PRIVACY.txt file in the root directory? I can integrate it here soon after taking a closer look at the Scribe implementation.

**Comment by andrewtavis** [comment]
Hey @wldeh 👋 Yes, the `PRIVACY.txt` file would go in the project root. If you have interest in integrating it, then please do 😊 I’m very focused on the v2.2.0 release of Scribe at the moment, but my priority will switch hard back to activist next week :) Really grateful to get some help here!

**Comment by wldeh** [comment]
> Hey @wldeh 👋 Yes, the `PRIVACY.txt` file would go in the project root. If you have interest in integrating it, then please do 😊 I’m very focused on the v2.2.0 release of Scribe at the moment, but my priority will switch hard back to activist next week :) Really grateful to get some help here!

Completely understood. Alright yeah I'll definitely look to have a polished setup here by next week. Thanks again!

**Comment by andrewtavis** [comment]
Thank you!! 😊

File: frontend/pages/legal/privacy-policy.vue [hunk 1]
```diff
@@ -1,14 +1,147 @@
...
+        <section>
+          <h2 class="text-2xl font-semibold mb-4">Introduction</h2>
+          <p>
+            At Activist.org, we value your privacy and are committed to
+            maintaining the trust and confidence of our users. This Privacy
+            Policy explains how we handle your information and the choices you
+            have regarding your information.
+          </p>
+          <p>
+            By using Activist.org, you agree to the provisions of this Privacy
```
Comment (andrewtavis) [inline frontend/pages/legal/privacy-policy.vue:19]: activist is always lower case 🙃

File: frontend/pages/legal/privacy-policy.vue [hunk 2]
```diff
@@ -1,14 +1,147 @@
...
+          </p>
+        </section>
+        <section>
+          <h2 class="text-2xl font-semibold mb-4">
+            Data Sharing and Third-Party Services
+          </h2>
+          <p>
+            We do not share your personal information with any third parties,
+            nor do we use any third-party services that collect or process
+            personal information. Activist.org is dedicated to ensuring the
```
Comment (andrewtavis) [inline frontend/pages/legal/privacy-policy.vue:67]: Same here 🙃

File: frontend/pages/legal/privacy-policy.vue [hunk 3]
```diff
@@ -1,14 +1,147 @@
...
+          <h2 class="text-2xl font-semibold mb-4">Your Rights</h2>
+          <p>
+            As we do not collect personal information, you do not need to worry
+            about exercising any rights related to your personal data. However,
+            if you have any questions or concerns about this Privacy Policy or
+            our practices, please feel free to contact us.
+          </p>
+          <p>
+            We are committed to transparency and user control. We encourage
+            users to contribute to Activist.org and participate in discussions
```
[the remark this answers is not in the document]
Comment (andrewtavis) [reply frontend/pages/legal/privacy-policy.vue:106]: And one below :)

File: frontend/pages/legal/privacy-policy.vue [hunk 4]
```diff
@@ -1,14 +1,147 @@
...
+            and ensure that it remains a safe, inclusive, and empowering
+            platform for all.
+          </p>
+        </section>
+        <section>
+          <h2 class="text-2xl font-semibold mb-4">Changes to This Policy</h2>
+          <p>
+            We may update this Privacy Policy from time to time. If we make
+            significant changes, we will notify you through the platform or by
+            other means. By continuing to use Activist.org after any changes are
```
Comment (andrewtavis) [inline frontend/pages/legal/privacy-policy.vue:118]: Last one!!! 😊

**Review by andrewtavis — CHANGES_REQUESTED** [review]
This reads really well, @wldeh! I'll go through and do some edits on my side, but from my end we can just change the `Activist.org` mentions to `activist.org`. I can then go through and format the page like https://activist.org/en/about/. 

You're also welcome to commit an updated version as a `PRIVACY.txt` file! 

Thanks again, @wldeh! 🚀 Super helpful 😊

**Comment by wldeh** [comment]
> Alsooo, if you're a Matrix user, feel free to join us in the [public chat](https://matrix.to/#/#activist_community:matrix.org) so we can connect on future issues 😊

Thank you for the link! I'm actually not a Matrix user yet, but I'm interested in learning more about it. Going to setup an account here shortly.

**Comment by andrewtavis** [comment]
Great, @wldeh! I'm almosttt done with the Scribe release, btw 😊 So much to do for this one as the data process changed dramatically. Will get to this soon after :)

Feel free to send along the changes that I mentioned [here](https://github.com/activist-org/activist/pull/95#pullrequestreview-1384842984) and we can merge directly :) :)

**Comment by wldeh** [comment]
> This reads really well, @wldeh! I'll go through and do some edits on my side, but from my end we can just change the `Activist.org` mentions to `activist.org`. I can then go through and format the page like https://activist.org/en/about/.
> 
> You're also welcome to commit an updated version as a `PRIVACY.txt` file! We can then merge after that :)
> 
> Thanks again, @wldeh! 🚀 Super helpful 😊



> This reads really well, @wldeh! I'll go through and do some edits on my side, but from my end we can just change the `Activist.org` mentions to `activist.org`. I can then go through and format the page like https://activist.org/en/about/.
> 
> You're also welcome to commit an updated version as a `PRIVACY.txt` file! We can then merge after that :)
> 
> Thanks again, @wldeh! 🚀 Super helpful 😊

Thank you for your feedback, @andrewtavis! I fixed all the Activist.org issues here and created a PRIVACY.txt file in the root directory. Ill try to push a commit here in the next day that replicates the 

> Great, @wldeh! I'm almosttt done with the Scribe release, btw 😊 So much to do for this one as the data process changed dramatically. Will get to this soon after :)
> 
> Feel free to send along the changes that I mentioned [here](https://github.com/activist-org/activist/pull/95#pullrequestreview-1384842984) and we can merge directly :) :)

Alright yeah I fixed all the Activist.org issues here and created a PRIVACY.txt file in the root directory. Ill try to push a commit here in the next day that replicates the about page as well. Thanks again for your feedback, @andrewtavis and apologies for the delay!

**Comment by wldeh** [comment]
> No delay whatsoever, @wldeh! The stuff about formatting the page and everything is definitely beyond what I'd expect 😊👏 Thank you! What you can do is just use the about pages icon and everything, so just copy that page and replace the content with the policy. I'll then go in and replace the activist icon with something like a shield or something like we have for Scribe :)

Perfect Alright. Just pushed these changes. Things are not looking the cleanest but feel free to review when you can.

**Review by andrewtavis — APPROVED** [review]
Thank you, @wldeh! Sorry for getting to this a bit late :) Has been a crazy busy week for me, but I'm shifting focus back to activist going forward 😊 I'll do some minor edits to the text and switch over the privacy policy icon to something to a shield with a checkmark so we're telling people they're safe.

Looking forward to further collaborations!

**Comment by wldeh** [comment]
> Thank you, @wldeh! Sorry for getting to this a bit late :) Has been a crazy busy week for me, but I'm shifting focus back to activist going forward 😊 I'll do some minor edits to the text and switch over the privacy policy icon to something to a shield with a checkmark so we're telling people they're safe.
> 
> Looking forward to further collaborations!

Great to hear Andrew and no worries at all! The checkmark definitely makes more sense. Happy to say I am a fan of this projects vision and definitely planning on making some more significant contributions here moving forward. Cheers!

**Comment by andrewtavis** [comment]
Wonderful to hear 😊😊

**Pull request merged**